jfrog_platform_cloud/assets/logs/jfrog_artifactory_tests.yaml

```diff
@@ -1,234 +1,223 @@
-          "return_status" : "200",
-          "username" : "non_authenticated_user"
-        },
-        "ddsource" : "jfrog_artifactory"
-      }
-    service: "jfrog.saas.rt.artifactory.request"
-    tags:
-     - "source:LOGS_SOURCE"
-    timestamp: 1700008877896
-    trace_id: "b05db4d5ae4fc51a"
- -
-  sample: |-
-    {
-      "tenant_id" : "b0r4mbvkakcf3",
-      "hostname" : "jfrog-cloud",
-      "ddtags" : "owner:partnershipstag1",
-      "service" : "jfrog.saas.rt.artifactory.request",
-      "message" : {
-        "log_timestamp" : "2023-11-15T00:41:37.857Z",
-        "trace_id" : "d4328165e6fe6921",
-        "remote_address" : "127.0.0.1",
-        "request_method" : "GET",
-        "source" : "jfrog_artifactory",
-        "response_content_length" : "-1",
-        "request_url" : "/api/v1/system/liveness",
-        "request_user_agent" : "curl/7.76.1",
-        "request_content_length" : "0",
-        "request_duration" : "1",
-        "log_source" : "jfrog.saas.rt.artifactory.request",
-        "return_status" : "200",
-        "username" : "non_authenticated_user"
-      },
-      "ddsource" : "jfrog_artifactory"
-    }
-  result:
-    custom:
-      ddsource: "jfrog_artifactory"
-      ddtags: "owner:partnershipstag1"
-      duration: "1"
-      host: "jfrog-cloud"
-      http:
-        method: "GET"
-        status_code: "200"
-        url: "/api/v1/system/liveness"
-        useragent: "curl/7.76.1"
-      jfrog:
-        cloud:
-          request_content_length: "0"
-          response_content_length: "-1"
-          tenant_id: "b0r4mbvkakcf3"
-      log_source: "jfrog.saas.rt.artifactory.request"
-      log_timestamp: "2023-11-15T00:41:37.857Z"
-      network:
-        client:
-          ip: "127.0.0.1"
-      service: "jfrog.saas.rt.artifactory.request"
-      source: "jfrog_artifactory"
-      usr:
-        id: "non_authenticated_user"
-    message: |-
-      {
-        "tenant_id" : "b0r4mbvkakcf3",
-        "hostname" : "jfrog-cloud",
-        "ddtags" : "owner:partnershipstag1",
-        "service" : "jfrog.saas.rt.artifactory.request",
-        "message" : {
-          "log_timestamp" : "2023-11-15T00:41:37.857Z",
-          "trace_id" : "d4328165e6fe6921",
-          "remote_address" : "127.0.0.1",
-          "request_method" : "GET",
-          "source" : "jfrog_artifactory",
-          "response_content_length" : "-1",
-          "request_url" : "/api/v1/system/liveness",
-          "request_user_agent" : "curl/7.76.1",
-          "request_content_length" : "0",
-          "request_duration" : "1",
-          "log_source" : "jfrog.saas.rt.artifactory.request",
-          "return_status" : "200",
-          "username" : "non_authenticated_user"
-        },
-        "ddsource" : "jfrog_artifactory"
-      }
-    service: "jfrog.saas.rt.artifactory.request"
-    tags:
-     - "source:LOGS_SOURCE"
-    timestamp: 1700008897857
-    trace_id: "d4328165e6fe6921"
+  - sample: >-
```

Comment:
there is an issue generating the result.custom data, it might becoming from this character. Try replacing by |-